.github/workflows/build.yaml

```diff
@@ -66,6 +66,14 @@ jobs:
               run: |
                   mkdir /tmp/cores || true
                   sysctl -w kernel.core_pattern=/tmp/cores/core.%u.%p.%t || true
+
+            - name: Bootstrap cache
+              uses: actions/cache@v3
+              with:
+                  key: ${{ runner.os }}-env-${{ hashFiles('scripts/setup/*') }}
```

Comment:
bootstrap also depends on pigweed version. Is there some way to include that in the cache logic?

Reply:
I've added pigweed repo dir files as a source for the hash: `hashFiles('scripts/setup/*', 'third_party/pigweed/**')`